src/SDL_renderer_textengine.c

```diff
@@ -838,10 +852,12 @@ TTF_TextEngine *TTF_CreateRendererTextEngine(SDL_Renderer *renderer)
         return NULL;
     }
 
+    int atlas_texture_size = SDL_GetNumberProperty(props, TTF_PROP_RENDERER_TEXT_ENGINE_ATLAS_TEXTURE_SIZE, 512);
```

Comment:
Do we not need a sanity check here, e.g. against `atlas_texture_size > 0` ?

Reply:
Adding an `(int)` cast here would be good here to silence MSVC

Reply:
I assume it's intentional that you reduced default atlas size from 1024 to 512?

Reply:
> I assume it's intentional that you reduced default atlas size from 1024 to 512?

Yes, I think now that we can set the size, a smaller default might be better.

Reply:
Let's leave the default to 1024. Now that we can set the size, people can make it smaller if they want. :)

Reply:
Reverted